apps/dot-cdn/src/app/app.module.ts

```diff
@@ -17,7 +22,11 @@ import { HttpClientModule } from '@angular/common/http';
         DropdownModule,
         BrowserAnimationsModule,
         HttpClientModule,
-        RouterModule.forRoot([])
+        RouterModule.forRoot([]),
+        TabViewModule,
+        ChartModule,
```

Comment:
not sure if this is the right place to import those...

Reply:
Can you explain why? We're only using one module for this app.

Reply:
sorry @ivorpad my bad, ignore my comment! :P